src/App.js

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
+import Header from "./components/Header";
```

Comment:
Each component should better be placed in a separate component folder, all the nested components should be placed here as well (in its own separate component folders).